src/main/webapp/app/pages/newsPage/ChangedAnnotatonListItem.tsx

```diff
@@ -55,6 +56,16 @@ export const ChangedAnnotationListItem = (props: {
       break;
   }
 
+  const geneColumnIndex = getGeneColumnIndex(annotationColumnHeader);
+
+  // transform the gene input to link(s)
+  props.data.forEach(row => {
+    const geneInput = row.content[geneColumnIndex].content;
```

Comment:
What happens if `geneColumnIndex` is -1?

Reply:
I think all types have `gene` in it, but i guarded with `geneColumnIndex > -1` in the new update